packages/CodingStandard/tests/Fixer/Property/ArrayPropertyDefaultValueFixer/correct/correct.php.inc

```diff
@@ -0,0 +1,9 @@
+<?php
+
+class SomeClass
+{
+	/**
+	 * @var ArrayObject|int[]
+	 */
+	public $property = [];
```

Comment:
I think ` = []` should be removed. Otherwise it's false positive.

Reply:
Now that you mention it, yes, obviously. That fixed the problem. :-)